client/main.coffee

```diff
@@ -608,6 +609,18 @@ class Selection
     for id of @selected
       return true
     false
+  delete: ->
+    return unless @nonempty()
+    ## The following is similar to eraser.up:
+    undoableOp
+      type: 'multi'
+      ops:
+        for obj in @ids()
```

Comment:
this wants to be `for id in @ids()`, right?

Reply:
Clearly I didn't test yet... 🙂 Thanks for reading over the changes!